### PR TITLE
feat(VBC-120): guest wishlist - isInWishlist action

### PR DESCRIPTION
**Pull request by WojtekTheWebDev**
Add isInWishlist action to useGuestWishlist

## Description

Add isInWishlist action to useGuestWishlist

## Related Issue

VBC-120

## Motivation and Context

User should be able to check if the item is in the wishlist

## How Has This Been Tested?

Unit tests

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by scernisevs — APPROVED** [review]
LGTM

File: packages/composables/src/useGuestWishlist/actions/isInWishlist.ts [hunk 1]
```diff
@@ -1,3 +1,14 @@
-export const isInWishlist = (): any => {
+import { BIGCOMMERCE_GUEST_WISHLIST_ID } from '../../helpers/consts';
+import { GuestWishlist } from '../../types';
 
+export const isInWishlist = (productId: number, variantId?: number): boolean => {
+  const localStorageItem = localStorage.getItem(BIGCOMMERCE_GUEST_WISHLIST_ID);
+
+  if (!localStorageItem) {
+    return false;
+  }
```
Comment (zoltantakacs-born) [inline packages/composables/src/useGuestWishlist/actions/isInWishlist.ts:9]: I think it should not load the wishlist again, just use the already loaded data. You should pass the currentWishlist to this function from the factory. (You can also check the useCartFactory if it is not clear)

**Review by zoltantakacs-born — CHANGES_REQUESTED** [review]

File: packages/composables/src/useGuestWishlist/actions/isInWishlist.ts (hunk 1 above)
Comment (WojtekTheWebDev) [reply packages/composables/src/useGuestWishlist/actions/isInWishlist.ts]: ty @zoltantakacs-born, changed.

File: packages/composables/__tests__/useGuestWishlist/isInWishlist.spec.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    const res = await isInWishlist(guestWishlistMock, nonExistingItem);
+
+    expect(res).toBe(false);
+  });
+
+  it('should return false if there product is not in the wishlist', async () => {
+    const res = await isInWishlist(guestWishlistMock, nonExistingItem);
+
+    expect(res).toBe(false);
+  });
```
Comment (zoltantakacs-born) [inline packages/composables/__tests__/useGuestWishlist/isInWishlist.spec.ts:42]: duplicated

**Review by zoltantakacs-born — APPROVED** [review]

**Pull request merged**